README.md

```diff
@@ -60,37 +61,45 @@ Cafe Nero Finikoudes, Larnaka
 to structured, usable data:
 
 ```json
-[
-  {
-    "id": "event-description",
+{
+  "event-description": {
+    "order": 0,
     "title": "Event Description",
     "text": "Let's meet for coffee and chat about tech, coding, Cyprus and the newly formed\nCDC (Cyprus Developer Community)."
   },
-  {
-    "id": "location",
+  "location": {
+    "order": 1,
     "title": "Location",
     "text": "Cafe Nero Finikoudes, Larnaka"
   },
-  {
-    "id": "date",
+  "date": {
+    "order": 2,
     "title": "Date",
     "text": "11.03.2022",
     "date": "2022-03-11"
   },
-  { "id": "time", "title": "Time", "text": "16:00\n", "time": "16:00" }
-]
+  "time": {
+    "title": "Time",
+    "text": "16:00",
+    "time": "16:00"
+  }
```

Comment:
I imagine this would also have a `"order"` field and was just missed in updating the rest of the docs